manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -124,6 +128,10 @@ spec:
             - name: shared-doca-driver-toolkit
               mountPath: /mnt/shared-doca-driver-toolkit
             {{- end}}
+            {{- if .CrSpec.DriverInventoryPath }}
```

Comment:
Is it needed for the DTK container?

Reply:
Per discussion with Vlad, the main ofed container gathers the build artifacts from the DTK container. The current idea is to let the main container handle the inventory

Reply:
So no need for that mount on the DTK container

Reply:
Yep